### ISSUE TITLE
PokeMMO crashed

### ISSUE THREAD

**Issue by ji185013** [MrJackSpade/RG35XXP-XFCE#6]
PokeMMO crashed when clicking select button for ROM setup. Any idea how to fix it?

**Comment by MrJackSpade** [comment]
I don't see anything about an ARM build of PokeMMO on the website, are you using an ARM build or building from scratch?

**Comment by Vulpezzz** [comment]
Hey awesome project!! There is a version of pokemmo for arm, is the protable version. Check it out: https://forums.pokemmo.com/index.php?/topic/172812-pokemmo-compiled-for-arm/ . I tried to install jdk 17 and pokemmo in other os, however because it didnt used x11 i had no success. I came along your project while searching a way to install an actual distro in this to try to play it. It seems that thanks to your amazing work i will be able to run it. Or is that something that makes it impossible to run?
